packages/antify-admin/src/pages/admin/[tenantId]/users/[userId].vue

```diff
@@ -20,62 +18,96 @@ const { data: roles } = await useFetch(
 
 const { $toaster } = useNuxtApp();
 const errors = ref([]);
+const search = ref('');
 const loading = ref<Boolean>(false);
 const validator = ref(baseValidator);
 
-const onSubmit = async () => {
+const roleOptions = computed(() => {
+  return (
+    roles.value as {
+      permissions: string[];
+      name: string;
+      id: string;
+      isAdmin: boolean;
+    }[]
+  ).map((role) => ({
+    value: role.id,
+    label: role.name,
+  }));
+});
+
+async function onSubmit() {
   loading.value = true;
   errors.value = [];
 
-  validator.value.validate(data.value.default, 1);
+  validator.value.validate(user.value.default, 1);
 
   if (validator.value.hasErrors()) {
     loading.value = false;
+
     return;
   }
 
+  await useFetch<PutResponse>(`/api/users/${useRoute().params.userId}`, {
+    ...useDefaultFetchOpts(),
+    ...{
+      method: 'PUT',
+      body: user.value.default,
+    },
+  });
+  loading.value = false;
+
+  $toaster.toastUpdated();
+  refresh();
+}
+
+async function banUser() {
   const { data: response } = await useFetch<PutResponse>(
-    `/api/users/${useRoute().params.userId}`,
+    `/api/users/ban/${useRoute().params.userId}`,
```

Comment:
Please change api url to /api/users/[userId]/ban and */unban to keep the logic: app url === api url